src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -423,10 +423,12 @@ final case class TenantApiServiceImpl(
           )
       }
       targetTenant        <- tenantManagementService.getTenantById(targetTenantUuid).map(_.toManagement)
-      attribute = targetTenant.attributes.flatMap(_.certified).find(_.id == seed.id)
-      updatedTenant <- attribute.fold(
+      certifiedAttribute = targetTenant.attributes.flatMap(_.certified).find(_.id == seed.id)
+      updatedTenant <- certifiedAttribute.fold(
         tenantManagementService.addTenantAttribute(targetTenantUuid, seed.toCreateDependency(now))
-      )(_ => Future.failed(CertifiedAttributeAlreadyExists(targetTenantUuid, seed.id)))
+      )(_ =>
+        tenantManagementService.updateTenantAttribute(targetTenantUuid, attribute.id, attribute.toCertifiedSeed(now))
```

Comment:
@beetlecrunch should we use the same approach as the revocation and fail if the attribute is already assigned but not revoked?

Reply:
@galales can you check this point again?